data/World/Dodongos Cavern.json

```diff
@@ -19,6 +19,30 @@
             "Dodongos Cavern GS Scarecrow": "
                 (can_use(Scarecrow) or can_use(Longshot) or (logic_dc_scarecrow_gs and (is_adult or
                 (can_child_attack and (can_blast_or_smash or Progressive_Strength_Upgrade)))))",
+
+            "Dodongos Cavern Lower Lizalfos Hidden Recovery Heart": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Lower Lizalfos Pot 1" : "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Lower Lizalfos Pot 2" : "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Lower Lizalfos Pot 3" : "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Lower Lizalfos Pot 4" : "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 1": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 2": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 3": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 4": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 5": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
+            "Dodongos Cavern Right Side Pot 6": "here(can_blast_or_smash or Progressive_Strength_Upgrade) or is_adult",
```

Comment:
For the DC right side hallways and 1st Lizalfos fight rooms, is_adult or has_explosives or Progressive_Strength_Upgrade is what you want. Since adult can just collect all of it for free, there's no reason to worry about him opening anything up for child, so the here() and also the hammer check as part of blast/smash is not doing anything.